src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -5358,3 +5364,6 @@
           NetworkCommissioning:
               WiFiSecurity:
                   WEP: WepPersonal
+                  WPAPersonal: WpaPersonal
```

Comment:
maybe WPA, WPA2 on the left side or change WEP to WERPersonal on the leftto be consistent.

Reply:
No, this part is right.  This is the "renames" section, and it shows the new name on the left, the old name on the right.  The spec names are "WEP", "WPA-PERSONAL", "WPA2-PERSONAL", etc, so our API names end up being "WEP", "WPAPersonal", "WPA2Personal", etc.

Reply:
I see that makes sense. i thought it was some labels. Thanks for the clarification